{"pageProps":{"page":{"slug":"class.closedgeneratorexception.md","content":"# Клас ClosedGeneratorException\n\n(PHP 5 >= 5.5.0, PHP 7, PHP 8)\n\n## Вступ\n\nИсключение**ClosedGeneratorException** викидається при спробі отримати значення із закритого ітератора, реалізованого класом [Generator](class.generator.md)\n\n## Огляд класів\n\n```classsynopsis\n\n    \n     class ClosedGeneratorException\n    \n\n    \n     extends\n      Exception\n     {\n\n    /* Наследуемые свойства */\n    \n      protected\n      string\n       $message = \"\";\nprivate\n      string\n       $string = \"\";\nprotected\n      int\n       $code;\nprotected\n      string\n       $file = \"\";\nprotected\n      int\n       $line;\nprivate\n      array\n       $trace = [];\nprivate\n      ?Throwable\n       $previous = null;\n\n\n    /* Наследуемые методы */\n    \n   public Exception::__construct(string $message = \"\", int $code = 0, ?Throwable $previous = null)\n\n    final public Exception::getMessage(): string\nfinal public Exception::getPrevious(): ?Throwable\nfinal public Exception::getCode(): int\nfinal public Exception::getFile(): string\nfinal public Exception::getLine(): int\nfinal public Exception::getTrace(): array\nfinal public Exception::getTraceAsString(): string\npublic Exception::__toString(): string\nprivate Exception::__clone(): void\n\n   }\n```\n","title":"Клас ClosedGeneratorException","contentType":2,"navigation":[{"errorexception.getseverity.md":"« ErrorException::getSeverity"},{"class.error.md":"Error »"},{"index.md":"PHP Manual"},{"reserved.exceptions.md":"Обумовлені винятки"}]}},"__N_SSG":true}